stream-transformation-test/stream-transformation-it/src/test/java/uk/gov/justice/framework/tools/transformation/StreamTransformationIT.java

```diff
@@ -98,6 +107,12 @@ private boolean eventStoreTransformedEventPresent() throws SQLException {
 
     private void insertEventLogData() throws SQLException, InvalidSequenceIdException {
```

Comment:
Not sure if it would be cleaner if we had a method to insert event for stream, at version with name. But this is ok.

Reply:
Done, refactored method to have parameters and removed added method.